_data/people.yml

```diff
@@ -47,6 +47,9 @@
 - group: 'Undergraduate Students'
   people:
   - name: "Bowen Quan"
+  - name: "Gerson A. Esquivel Garcia"
+    url: "gaeg"
+    image: Gerson-3 copy.jpg
```

Comment:
These lines should be a new entry, ideally after Damian's below

Reply:
Fixed, I did not see the file was had text beyond what the screen showed.